app/experimenter/nimbus-ui/README.md

```diff
@@ -0,0 +1,29 @@
+# Mozilla Experimenter Nimbus UI
+
+## Development
+
+- `yarn start` to start the app in development mode on `http://localhost:3000`
+- `yarn build` to create a production build
+- `yarn test` to run unit tests
+
+## Testing
+
+This package uses [Jest](https://jestjs.io/) to test its code. By default `yarn test` will test all JS files under `src/`.
+
+Test specific tests with the following commands:
+
+```bash
+# Test for the component AppLayout
+yarn test AppLayout
+
+# Grep for "renders as expected"
+yarn test -t="renders as expected"
+```
+
+You can also see the test coverage with details by running the following command:
+
+```
+yarn test --coverage
+```
+
+Refer to Jest's [CLI documentation](https://jestjs.io/docs/en/cli) for more advanced test configuration.
```

Comment:
Maybe it can go in this README or the root README, but I think we should document how this app should be used in relation to the rest of the codebase (for example `yarn`ing from the `/app` dir, using `make check`, or any other weird quirks). What do you think?

Reply:
I think mentioning `yarn`ing from `app/` makes sense in the root README, though `make check` is mentioned in https://github.com/mozilla/experimenter/blob/main/contributing.md. We could probably use a quick explainer of `nimbus-ui`/`legacy-ui` in the root and link to the ADR if that'd be useful, or maybe just a few sentences after "Experimenter is a platform for managing experiments in Mozilla Firefox." to expound upon that and make getting started more clear. Worth filing an issue for?